go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/golang/protobuf v1.5.3
 	github.com/gorilla/websocket v1.5.1
 	github.com/grafana/xk6-browser v1.2.1
+	github.com/grafana/xk6-dashboard v0.6.1
```

Comment:
You here use the latest tagged version, but you also dropped a bunch of dependancies and other changes after that.

Can we please either:
1. tag a new release and use that
2. just use the commit for now

Reply:
as I wrote
> This is not yet the final version of the integration. The goal is to integrate xk6-dashboard v0.7.0, but that release does not exist yet (waiting for dependencies).

from integration point of view there is no difference (expect dependencies), so this is why I just used latest tagge version.... I'll update to v0.7.0 when it will ready

Reply:
Done, I created a pre-release.